### DISCUSSION TITLE
کانفیگ ایرانسل

### DISCUSSION THREAD

**Discussion by ituser96** [Gozargah/Marzban#477] [General]
سلام دوستان؛

یک کانفیگ برای ایرانسل میشه لطف کنید؟
من با کانفیگ های پیشفرض با مخابرات و شاتل و همراه اول ... تست کردم سرعت عالیه،
ولی با ایرانسل کند هست.
راهکاری دارید کانفیگی انجام بدیم برای ایرانسل؟
تشکر

**Comment by pooyahpx** [comment]
از پورت 443 استفاده کن

**Comment by ituser96** [comment]
من از امروز صبح فقط با اینترنت مخابرات اوکی هستم،
با بقیه نت ها یا متصل نمیشه هیچ کانفیکی یا خیلی خیلی کند هست.
برای شما هم همینطور شده؟

**Reply by pooyahpx** [reply]
نه مشکل از ایپی خودتونه بهتره برای اینکه ایپیاتون بلاک نشه تونل کنید ای پی vps رو و بهتره به ایران تونل کنید و از دامین بجای ای پی خالی استفاده کنید

**Comment by jo0oj0** [comment]
شما برای پهنای باند ایران چیکار میکنید؟
مثلا من یه سرور VPS معمولی از ایران هم بگیرم پهنای باندش خیلی کم هست. اگه بخوام شخصی هم استفاده کنم با پهنای باند تعرفه معمولی 100 گیگ که داره، خیلی خیلی کم میارم اونم با قیمت ماهی حداقل 200تومن ، چه برسه بخوام به 2-3 نفر دیگه هم دسترسی استفاده بدم.
اگه بخوام سرور با پهنای باند بالاتر بردارم که کلا هزینه ها میره بالا، از یه طرف هزینه سرور خارج + از یه طرف سرور داخل.

**Reply by pooyahpx** [reply]
شما میتونید به همه جا تانل بزنید حتما نیاز نیست که ایران باشه ولی برای اینکه شک روی ایپیتون به وجود نیاد بهترین راهه کار دیگه ای که میتونید بکنید اینه که سایتای ایرانیو ببندین و به یه vps مثل ترکیه یا هرجای دیگه تانل کنید

**Comment by ituser96** [comment]
با چه روشی تانل میزنید از ایران به خارج؟ کدوم بهتره  iptable یا ghost ؟

**Reply by pooyahpx** [reply]
https://github.com/Gozargah/Marzban/discussions/473

**Comment by sinashk** [comment]
سلام دوستان 
سرور من هم امروز از کار افتاد
پنل یکی در میون باز میشه 
هیچ کدوم از کانفیگ ها هم کار نمی کنن
پیینگ آی پی روهم گرفت که متاسفانه لاست میده
به نظرتون تونل کنم یا سرور جدید بگیرم 1 هفته مونده تا تموم شه؟

**Reply by pooyahpx** [reply]
اگه ایپیتون فیلتر شده بهتره کلا یدونه vps جدید تهیه کنید و بعدش تانل کنید چون الان تانل کردنتون فایده نداره

**Reply by sinashk** [reply]
آیپی روی نت همراه اول اوکی بود ولی روی نت ایرانسل و آسیاتک کلا قطع بود
بعد 24 ساعت ساعت اتوماتیک باز شد آی پی و دوباره همه کانفیگا کار می کنن
عجیبب بود علتش و نفهمیدم چیه ؟!

**Reply by pooyahpx** [reply]
اگه کانفیگتون رو بقیه نتا کار میکنه و رو بعضی از نتا جواب نمیده کانفیگای جدید با flow و با xtls بسازین

**Reply by teststar121** [reply]
مثلا چه کانفیگی ؟ ریلیتی خوبه ؟
یا vless+xtls+tcp چی ؟

**Comment by ituser96** [comment]
برای منم یه وقتایی رو همه نت ها چند لحظه یا چند ساعتی همه کانفیگ ها قطع میشن بعد وصل میشه همشون.
بنظرم اختلال هست. برادران مشغول فیلتر بازی هستن..
سایت های داخلی رو هم بستم کلا.
تنها کاری که کردم یه سرور ایران خریدم تانلش کردم. همین.
البته با اینکه تانل از ایران هست، بازم دیشب دچار قطعی شدم ولی بازم بعدش همه چی اوکی هست.

**Reply by pooyahpx** [reply]
اگه رو سرور از dest درست استفاده کنید هیچ موقع اختلال به وجود نمیاد از پورت 8080 و با dest خوب به عنوان مثال efmspd2.mci.ir:8080 و سرور www.speedtest.net استفاده کنید به دلیل اینکه امکان فیلترشون وجود نداره و همه ی نتا وصل میشه هیچ مشکلی براتون به وجود نمیاد ❤️
